package/expo-package/package.json

```diff
@@ -17,7 +17,6 @@
     "expo": ">=44.0.0",
     "expo-document-picker": "^9.1.2",
     "expo-file-system": "^11.0.2",
-    "expo-haptics": "^10.0.0",
```

Comment:
Why is this removed? We use it in the (`expo-package`)[https://github.com/GetStream/stream-chat-react-native/blob/develop/package/expo-package/src/index.js#L8]

Reply:
That's a mistake, thanks for spotting it